src/ansys/mapdl/core/krylov.py

```diff
@@ -405,7 +405,7 @@ def solve(self, freq_start, freq_end, freq_steps, ramped_load=True):
 
         Returns
         -------
-        AnsMat
+        AnsMath
```

Comment:
The `AnsMat` mention was, in my opinion, not a typo. 
The output is supposed to be a matrix.

Reply:
This is valid for all the `AnsMath` mentions in this PR.
`AnsMat` and `AnsVec` are `AnsMath` objects.